test/HealthChecks.EventStore.gRPC.Tests/DependencyInjection/RegistrationTests.cs

```diff
@@ -0,0 +1,39 @@
+namespace HealthChecks.EventStore.gRPC.Tests.DependencyInjection
```

Comment:
Please use file-scoped namespaces for all new code.

Reply:
With pleasure 😄